docs/interpreter/cassandra.md

```diff
@@ -793,7 +812,17 @@ Below are the configuration parameters and their default value.
 
 <hr/>
 
-## 14. Bugs & Contacts
+## 14. Change Log
+
+**2.0** : 
```

Comment:
I think 1.0 and 2.0 versioning here could confusing while users can not know which Zeppelin version includes 1.0 and which Zeppelin version includes 2.0 of Cassandra interpreter.

At least mapping version with Zeppelin release version, like
```
2.0 - (zeppelin 0.5.6-incubating)
1.0 - (zeppelin 0.5.5-incubating)
```
would help user get idea about the version he is using.

Or just removing Cassandera interpreter version information  (1.0, 2.0) from this doc would also fine, while the Zeppelin documentation it-self published version by version
![image](https://cloud.githubusercontent.com/assets/1540981/12531432/37c760a2-c1af-11e5-958d-908e2eb7f04b.png)
It implies which features of Cassandera interpreters are release with which version of Zeppelin.